src/App.js

```diff
@@ -11,18 +11,18 @@ import DriverOperator from "./pages/crud-driver-operator/DriverOperator";
 import Roles from "./pages/roles & permissions/Roles";
 import ResetLink from "./pages/ResetLink";
 import ResetPwd from "./pages/ResetPwd";
-import Dashboard from "./pages/dashboard";
-
 import Assigned from "./pages/AssignDriver/Assigned";
 import UnAssigned from "./pages/AssignDriver/UnAssigned";
+import Dashboard from "./pages/Dashboard";
```

Comment:
this should be `import Dashboard from "./pages/dashboard";`

Reply:
Yes I did